addons/ZEN-vagrant-builder/installer/install-pf.sh

```diff
@@ -19,9 +19,9 @@ yum install perl -y
 yum install --enablerepo=$PFREPO $PFPACKAGE -y
 
 # Setting the hostname
-hostname PacketFence-ZEN
-echo "PacketFence-ZEN" > /etc/hostname
-echo "NETWORKING=yes HOSTNAME=PacketFence-ZEN" > /etc/sysconfig/network
+hostname PF-ZEN
```

Comment:
Shouldn't we just do that and leave the rest as-is?

Reply:
To make things cleaner and more consistent I changed everything but if you prefer the full name to be used elsewhere (like the VM ESX name), I can revert those

Reply:
That's my preference, but it doesn't mean we should do it!

I think "PF ZEN" isn't nice-looking for ESX name and others. For the hostname, heck I don't mind - we could even call it bob.

Reply:
Thinking about it again, `packetfence` is 12 characters which meets the requirement for the fix.

So I'll just change the hostname to that and leave the rest as is